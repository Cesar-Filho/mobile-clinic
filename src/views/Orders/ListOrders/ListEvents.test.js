import React from 'react';
import renderer from 'react-test-renderer';

import ListEvents from '.';

describe('<ListEvents />', () => {
  it('renderizar sem quebrar', () => {
    const rendered = renderer.create(<ListEvents />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
