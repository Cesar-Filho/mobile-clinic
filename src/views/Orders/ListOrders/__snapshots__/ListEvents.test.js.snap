// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListEvents /> renderizar sem quebrar 1`] = `
<RCTScrollView
  style={
    Array [
      Object {
        "height": 133.5,
        "paddingBottom": 40,
        "paddingLeft": 24,
        "paddingRight": 24,
        "paddingTop": 4,
        "width": 735,
      },
    ]
  }
>
  <View>
    <RCTScrollView
      data={
        Array [
          Object {
            "key": "a",
          },
          Object {
            "key": "b",
          },
          Object {
            "key": "c",
          },
        ]
      }
      disableVirtualization={false}
      getItem={[Function]}
      getItemCount={[Function]}
      horizontal={false}
      initialNumToRender={10}
      keyExtractor={[Function]}
      maxToRenderPerBatch={10}
      numColumns={1}
      onContentSizeChange={[Function]}
      onEndReachedThreshold={2}
      onLayout={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      stickyHeaderIndices={Array []}
      style={
        Array [
          Object {
            "display": "flex",
            "flexDirection": "column",
            "marginBottom": 24,
          },
        ]
      }
      updateCellsBatchingPeriod={50}
      viewabilityConfigCallbackPairs={Array []}
      windowSize={21}
    >
      <View>
        <View
          onLayout={[Function]}
          style={null}
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 14,
                  "marginBottom": 20,
                  "marginLeft": 0,
                  "marginRight": 0,
                  "marginTop": 20,
                },
              ]
            }
          >
            Quarta, 25 de Janeiro
          </Text>
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "borderLeftColor": "#733dbe",
                "borderLeftWidth": 4,
                "borderRadius": 5,
                "display": "flex",
                "elevation": 2,
                "flexDirection": "row",
                "height": 124,
                "marginBottom": 4,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 4,
                "opacity": 1,
                "paddingBottom": 16,
                "paddingLeft": 16,
                "paddingRight": 16,
                "paddingTop": 16,
                "shadowColor": "rgba(115, 61, 190, 0.1)",
              }
            }
          >
            <View
              accessibilityIgnoresInvertColors={true}
              style={
                Array [
                  Object {
                    "height": 92,
                    "marginRight": 12,
                    "width": 66,
                  },
                ]
              }
            >
              <Image
                source={
                  Object {
                    "uri": "https://s3-us-west-2.amazonaws.com/agendaedu-dev/schools/c5c1a933-cdef-4c9b-8a87-490f25c2538d/events/5380/attachments/1550866911-$1-original-poster-agendakids.jpeg",
                  }
                }
                style={
                  Array [
                    Object {
                      "bottom": 0,
                      "left": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                    },
                    Object {
                      "height": undefined,
                      "width": undefined,
                    },
                    undefined,
                  ]
                }
              />
            </View>
            <View
              style={
                Array [
                  Object {
                    "display": "flex",
                  },
                ]
              }
            >
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "justifyContent": "flex-start",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#999999",
                        "fontSize": 12,
                      },
                    ]
                  }
                >
                  EVENTOS
                </Text>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 2,
                      "flexShrink": 1,
                      "justifyContent": "center",
                    },
                  ]
                }
              >
                <View
                  style={
                    Array [
                      Object {},
                    ]
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": "#333333",
                          "fontSize": 16,
                        },
                      ]
                    }
                  >
                    Event 1
                  </Text>
                  <Text>
                    <Text />
                     
                    12:00
                  </Text>
                </View>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "justifyContent": "flex-end",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#999999",
                        "fontSize": 12,
                      },
                    ]
                  }
                >
                  Quarta, 25 de janeiro Ã s 20:00h
                </Text>
              </View>
            </View>
          </View>
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "borderLeftColor": "#733dbe",
                "borderLeftWidth": 4,
                "borderRadius": 5,
                "display": "flex",
                "elevation": 2,
                "flexDirection": "row",
                "height": 124,
                "marginBottom": 4,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 4,
                "opacity": 1,
                "paddingBottom": 16,
                "paddingLeft": 16,
                "paddingRight": 16,
                "paddingTop": 16,
                "shadowColor": "rgba(115, 61, 190, 0.1)",
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "display": "flex",
                  },
                ]
              }
            >
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "justifyContent": "flex-start",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#999999",
                        "fontSize": 12,
                      },
                    ]
                  }
                >
                  EVENTOS
                </Text>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 2,
                      "flexShrink": 1,
                      "justifyContent": "center",
                    },
                  ]
                }
              >
                <View
                  style={
                    Array [
                      Object {},
                    ]
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": "#333333",
                          "fontSize": 16,
                        },
                      ]
                    }
                  >
                    Event 2
                  </Text>
                  <Text>
                    <Text />
                     
                    16:00
                  </Text>
                </View>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "justifyContent": "flex-end",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#999999",
                        "fontSize": 12,
                      },
                    ]
                  }
                >
                  Quarta, 25 de janeiro Ã s 20:00h
                </Text>
              </View>
            </View>
          </View>
        </View>
        <View
          onLayout={[Function]}
          style={null}
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 14,
                  "marginBottom": 20,
                  "marginLeft": 0,
                  "marginRight": 0,
                  "marginTop": 20,
                },
              ]
            }
          >
            Quarta, 25 de Janeiro
          </Text>
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "borderLeftColor": "#733dbe",
                "borderLeftWidth": 4,
                "borderRadius": 5,
                "display": "flex",
                "elevation": 2,
                "flexDirection": "row",
                "height": 124,
                "marginBottom": 4,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 4,
                "opacity": 1,
                "paddingBottom": 16,
                "paddingLeft": 16,
                "paddingRight": 16,
                "paddingTop": 16,
                "shadowColor": "rgba(115, 61, 190, 0.1)",
              }
            }
          >
            <View
              accessibilityIgnoresInvertColors={true}
              style={
                Array [
                  Object {
                    "height": 92,
                    "marginRight": 12,
                    "width": 66,
                  },
                ]
              }
            >
              <Image
                source={
                  Object {
                    "uri": "https://s3-us-west-2.amazonaws.com/agendaedu-dev/schools/c5c1a933-cdef-4c9b-8a87-490f25c2538d/events/5380/attachments/1550866911-$1-original-poster-agendakids.jpeg",
                  }
                }
                style={
                  Array [
                    Object {
                      "bottom": 0,
                      "left": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                    },
                    Object {
                      "height": undefined,
                      "width": undefined,
                    },
                    undefined,
                  ]
                }
              />
            </View>
            <View
              style={
                Array [
                  Object {
                    "display": "flex",
                  },
                ]
              }
            >
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "justifyContent": "flex-start",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#999999",
                        "fontSize": 12,
                      },
                    ]
                  }
                >
                  EVENTOS
                </Text>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 2,
                      "flexShrink": 1,
                      "justifyContent": "center",
                    },
                  ]
                }
              >
                <View
                  style={
                    Array [
                      Object {},
                    ]
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": "#333333",
                          "fontSize": 16,
                        },
                      ]
                    }
                  >
                    Event 1
                  </Text>
                  <Text>
                    <Text />
                     
                    12:00
                  </Text>
                </View>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "justifyContent": "flex-end",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#999999",
                        "fontSize": 12,
                      },
                    ]
                  }
                >
                  Quarta, 25 de janeiro Ã s 20:00h
                </Text>
              </View>
            </View>
          </View>
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "borderLeftColor": "#733dbe",
                "borderLeftWidth": 4,
                "borderRadius": 5,
                "display": "flex",
                "elevation": 2,
                "flexDirection": "row",
                "height": 124,
                "marginBottom": 4,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 4,
                "opacity": 1,
                "paddingBottom": 16,
                "paddingLeft": 16,
                "paddingRight": 16,
                "paddingTop": 16,
                "shadowColor": "rgba(115, 61, 190, 0.1)",
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "display": "flex",
                  },
                ]
              }
            >
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "justifyContent": "flex-start",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#999999",
                        "fontSize": 12,
                      },
                    ]
                  }
                >
                  EVENTOS
                </Text>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 2,
                      "flexShrink": 1,
                      "justifyContent": "center",
                    },
                  ]
                }
              >
                <View
                  style={
                    Array [
                      Object {},
                    ]
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": "#333333",
                          "fontSize": 16,
                        },
                      ]
                    }
                  >
                    Event 2
                  </Text>
                  <Text>
                    <Text />
                     
                    16:00
                  </Text>
                </View>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "justifyContent": "flex-end",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#999999",
                        "fontSize": 12,
                      },
                    ]
                  }
                >
                  Quarta, 25 de janeiro Ã s 20:00h
                </Text>
              </View>
            </View>
          </View>
        </View>
        <View
          onLayout={[Function]}
          style={null}
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#999999",
                  "fontSize": 14,
                  "marginBottom": 20,
                  "marginLeft": 0,
                  "marginRight": 0,
                  "marginTop": 20,
                },
              ]
            }
          >
            Quarta, 25 de Janeiro
          </Text>
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "borderLeftColor": "#733dbe",
                "borderLeftWidth": 4,
                "borderRadius": 5,
                "display": "flex",
                "elevation": 2,
                "flexDirection": "row",
                "height": 124,
                "marginBottom": 4,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 4,
                "opacity": 1,
                "paddingBottom": 16,
                "paddingLeft": 16,
                "paddingRight": 16,
                "paddingTop": 16,
                "shadowColor": "rgba(115, 61, 190, 0.1)",
              }
            }
          >
            <View
              accessibilityIgnoresInvertColors={true}
              style={
                Array [
                  Object {
                    "height": 92,
                    "marginRight": 12,
                    "width": 66,
                  },
                ]
              }
            >
              <Image
                source={
                  Object {
                    "uri": "https://s3-us-west-2.amazonaws.com/agendaedu-dev/schools/c5c1a933-cdef-4c9b-8a87-490f25c2538d/events/5380/attachments/1550866911-$1-original-poster-agendakids.jpeg",
                  }
                }
                style={
                  Array [
                    Object {
                      "bottom": 0,
                      "left": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                    },
                    Object {
                      "height": undefined,
                      "width": undefined,
                    },
                    undefined,
                  ]
                }
              />
            </View>
            <View
              style={
                Array [
                  Object {
                    "display": "flex",
                  },
                ]
              }
            >
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "justifyContent": "flex-start",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#999999",
                        "fontSize": 12,
                      },
                    ]
                  }
                >
                  EVENTOS
                </Text>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 2,
                      "flexShrink": 1,
                      "justifyContent": "center",
                    },
                  ]
                }
              >
                <View
                  style={
                    Array [
                      Object {},
                    ]
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": "#333333",
                          "fontSize": 16,
                        },
                      ]
                    }
                  >
                    Event 1
                  </Text>
                  <Text>
                    <Text />
                     
                    12:00
                  </Text>
                </View>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "justifyContent": "flex-end",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#999999",
                        "fontSize": 12,
                      },
                    ]
                  }
                >
                  Quarta, 25 de janeiro Ã s 20:00h
                </Text>
              </View>
            </View>
          </View>
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "borderLeftColor": "#733dbe",
                "borderLeftWidth": 4,
                "borderRadius": 5,
                "display": "flex",
                "elevation": 2,
                "flexDirection": "row",
                "height": 124,
                "marginBottom": 4,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 4,
                "opacity": 1,
                "paddingBottom": 16,
                "paddingLeft": 16,
                "paddingRight": 16,
                "paddingTop": 16,
                "shadowColor": "rgba(115, 61, 190, 0.1)",
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "display": "flex",
                  },
                ]
              }
            >
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "justifyContent": "flex-start",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#999999",
                        "fontSize": 12,
                      },
                    ]
                  }
                >
                  EVENTOS
                </Text>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 2,
                      "flexShrink": 1,
                      "justifyContent": "center",
                    },
                  ]
                }
              >
                <View
                  style={
                    Array [
                      Object {},
                    ]
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": "#333333",
                          "fontSize": 16,
                        },
                      ]
                    }
                  >
                    Event 2
                  </Text>
                  <Text>
                    <Text />
                     
                    16:00
                  </Text>
                </View>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "flexBasis": 0,
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "justifyContent": "flex-end",
                    },
                  ]
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#999999",
                        "fontSize": 12,
                      },
                    ]
                  }
                >
                  Quarta, 25 de janeiro Ã s 20:00h
                </Text>
              </View>
            </View>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>
</RCTScrollView>
`;
